crates/query/src/scan/array_predicate.rs

```diff
@@ -409,4 +410,52 @@ impl ArrayPredicate for InList {
         let mask = sqd_polars::arrow::polars_boolean_to_arrow_boolean(&polars_mask);
         Ok(mask)
     }
-}
\ No newline at end of file
+}
+
+
+fn bitwise_and<const N: usize>(value: [u8; N], other: [u8; N]) -> [u8; N] {
+    let mut arr = [0; N];
+    for i in 0..N {
+        arr[i] = value[i] & other[i];
+    }
+    arr
+}
+
+
+pub struct BloomFilter {
+    bit_array: [u8; 64]
+}
+
+
+impl BloomFilter {
+    pub fn new<T: Hash>(values: Vec<T>) -> Self {
+        let mut bloom = sqd_bloom_filter::BloomFilter::<64>::new(7);
+        for value in values {
+            bloom.insert(&value);
```

Comment:
If you OR bits of all elements together here, you'll only be able to tell whether _all_ of them are present in the filter. But if we have to tell whether any of them is present, I'm afraid the only option is to have a list of masks, each one corresponding to a single account, and checking that `S_i & F == S_i` for any of them